mathicsscript/__main__.py

```diff
@@ -333,11 +335,6 @@ def main(
 
             evaluation = Evaluation(shell.definitions, output=TerminalOutput(shell))
             query, source_code = evaluation.parse_feeder_returning_code(shell)
-            if len(source_code) and source_code[0] == "!":
-                os.system(source_code[1:])
-                # Should we test exit code for adding to history?
```

Comment:
We could return `$Failure` or atleast print an error if the exit code isn't zero.